docs/csharp/language-reference/builtin-types/numeric-conversions.md

```diff
@@ -77,7 +77,7 @@ The following table shows the predefined explicit conversions between the built-
 
 Also note that:
 
-- When you convert a value of an integral type to another integral type, the result depends on the overflow [checking context](../keywords/checked-and-unchecked.md). In a checked context, the conversion succeeds if the source value is within the range of the destination type. Otherwise, an <xref:System.OverflowException> is thrown. In an unchecked context, the conversion always succeeds, and proceeds as follows:
+- When you convert a value of an integral type to another integral type, the result depends on the [overflow-checking context](../statements/checked-and-unchecked.md). In a checked context, the conversion succeeds if the source value is within the range of the destination type. Otherwise, an <xref:System.OverflowException> is thrown. In an unchecked context, the conversion always succeeds, and proceeds as follows:
```

Comment:
ping @tannergooding on this change. There are several locations in this content where we state that in a `checked` context, "an `OverflowException` occurs". Is this still always true, now that developers can write `checked` operators for their type? I'll tag the other locations I found.

If we should change the language, I'll suggest something like:

```suggestion
- When you convert a value of an integral type to another integral type, the result depends on the [overflow-checking context](../statements/checked-and-unchecked.md). In a checked context, the conversion succeeds if the source value is within the range of the destination type. Otherwise, an <xref:System.OverflowException> is thrown. for all built-in integral types. User defined numeric types define the behavior in a `checked` operator. In an unchecked context, the conversion always succeeds, and proceeds as follows:
```

Reply:
Thank you @BillWagner 
I don't think we need to change this text as this article is already about *built-in* numeric conversions between *built-in* types. Also, "user-defined numeric types" may be a niche use case => mentioning it can be more confusing. I would wait on user feedback to those articles.

As for the original queston:

>  Is this still always true, now that developers can write checked operators for their type?

The docs *recommend* that user-defined checked operators throw, but it's not guaranteed, so it's possible to observe no exception in case of overflow in a checked user-defined operator.

Reply:
I think I'd agree with Petr here. Even with user-defined types, there is no guarantee that "overflow" will happen. Some cases like `float` or `double` always "saturate" instead for example.

It really is type and input dependent.

Reply:
Thanks both. I'll mark this as resolved.